### PR TITLE
added wishlist

### PR DESCRIPTION
**Pull request by sushmita12345**
Added Wishlist page to the E-commerce website.

Sample
Screenshot 2022-02-11 at 1 59 08 AM

### CODE REVIEW

File: assets/css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+.rupee-icons {
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.card-price {
+  font-size: 1.4rem;
+  padding-top: 0.5rem;
```
Comment (PuneetP16) [inline assets/css/wishlist.css]: It's a known fact in NEOG now, to use pixels below 1rem. Surely could skip that if you have a use case.

**Review by PuneetP16 — APPROVED** [review]
Looks good.

File: assets/css/wishlist.css (hunk 1 above)
Comment (sushmita12345) [reply assets/css/wishlist.css]: Yes, I'll update.

**Review by palash23jain — APPROVED** [review]
color convention can be more consistent, but it is not a blocker.. As of now, it is good to go

**Pull request merged**